static/app/utils/platform.tsx

```diff
@@ -1,4 +1,33 @@
-import {mobile} from 'sentry/data/platformCategories';
+import {
+  backend,
+  desktop,
+  frontend,
+  mobile,
+  PlatformCategory,
+  serverless,
+} from 'sentry/data/platformCategories';
+
+export function platformToCategory(platform: string | undefined): PlatformCategory {
+  if (!platform) {
+    return PlatformCategory.other;
+  }
+  if (([...mobile] as string[]).includes(platform)) {
+    return PlatformCategory.mobile;
```

Comment:
you can just create a map of string to enum value and then look up enums instead

Reply:
something like `{ backend: PlatformCategory.backend }`

Reply:
Great suggestion, i'll give that a go!

Reply:
Hmm, this function is meant to map a platform like `javascript`, `javascript-react`, `node` etc, to its corresponding category, such as `backend`, `frontend`. We already have a readonly array of all the platforms in a category, i.e `frontend = ['javascript', 'javascript-react' ...]`. Maybe something like this is cleaner
```
 const platformMap = {};
  mobile.forEach(mobilePlatform => platformMap[mobilePlatform] = PlatformCategory.mobile)
  frontend.forEach(frontendPlatform => platformMap[frontendPlatform] = PlatformCategory.frontend)
...
return platformMap[platform]
  ```

Reply:
And then we can store it outside the function so it doesn't get recomputed everytime

Reply:
I pushed a change, let me know what you think, Thanks for the suggestions!